File: A5Chatbot-System/Chatbot.java
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class Chatbot {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Map<String, String> responses = new HashMap<>();

        responses.put("hello", "Hello! How can I assist you today?");
        responses.put("hi", "Hi there! How can I help you?");
        responses.put("how are you", "I'm just a bot, but I'm functioning perfectly!");
        responses.put("name", "I am a simple customer service chatbot.");
        responses.put("help", "Sure! You can ask me basic questions.");
        responses.put("thanks", "You're welcome! Happy to help.");
        responses.put("bye", "Goodbye! Have a great day!");

        System.out.println("====================================");
        System.out.println(" CUSTOMER SERVICE CHATBOT ");
        System.out.println("====================================");
        System.out.println("Chatbot: Hello! Type 'bye' to exit.\n");

        while (true) {
            System.out.print("You: ");
            String userInput = scanner.nextLine().toLowerCase().trim();

            if (userInput.equals("bye")) {
                System.out.println("Chatbot: " + responses.get("bye"));
                break;
            }

            boolean found = false;

            for (String key : responses.keySet()) {
                if (userInput.contains(key)) {
                    System.out.println("Chatbot: " + responses.get(key));
                    found = true;
                    break;
                }
            }

            if (!found) {
                System.out.println("Chatbot: Sorry, I didn't understand that. Please try again.");
            }
        }

        scanner.close();
    }
}

File: A5Chatbot-System/chatbot.py
# Customer Service Chatbot

responses = {
    "hello": "Hello! How can I assist you today?",
    "hi": "Hi there! How can I help you?",
    "how are you": "I'm just a bot, but I'm functioning perfectly!",
    "name": "I am a simple customer service chatbot.",
    "help": "Sure! You can ask me basic questions.",
    "thanks": "You're welcome! Happy to help.",
    "bye": "Goodbye! Have a great day!"
}

print("====================================")
print(" CUSTOMER SERVICE CHATBOT ")
print("====================================")
print("Chatbot: Hello! Type 'bye' to exit.\n")

while True:

    user_input = input("You: ").lower().strip()

    if user_input == "bye":
        print("Chatbot:", responses["bye"])
        break

    found = False

    for key in responses:

        if key in user_input:
            print("Chatbot:", responses[key])
            found = True
            break

    if not found:
        print("Chatbot: Sorry, I didn't understand that. Please try again.")